File: src/Adapter/src/Temporada.java
import java.util.ArrayList;

public class Temporada {
    private int numero;
    private ArrayList<Episodio> episodios;

    // Getters and Setters

    public int getNumero() {
        return numero;
    }

    public void setNumero(int numero) {
        this.numero = numero;
    }

    public ArrayList<Episodio> getEpisodios() {
        return episodios;
    }

    public void setEpisodios(ArrayList<Episodio> episodios) {
        this.episodios = episodios;
    }

    public ArrayList<Episodio> listarEpisodio() {
        return episodios;
    }
}

File: src/Adapter/src/models/temporada.ts
import Episodio from './episodio';

class Temporada {
    private numero: number = 0;
    private episodios: Episodio[] = [];

    // Getters and Setters
    public getNumero(): number {
        return this.numero;
    }

    public setNumero(numero: number): void {
        this.numero = numero;
    }

    public getEpisodios(): Episodio[] {
        return this.episodios;
    }

    public setEpisodios(episodios: Episodio[]): void {
        this.episodios = episodios;
    }

    public listarEpisodio(): Episodio[] {
        return this.episodios;
    }
}

export default Temporada;